app-typescript/components/Radio.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+
+interface IValue {
+    value: any;
+    text: string;
+    disabled?: boolean;
+}
+
+interface IProps {
+    model: any;
+    values: Array<IValue>;
+    side?: string;
```

Comment:
`labelSide` would be a more specific name. Which sides are supported? Use union types:
```labelSide: 'side1' | 'side2';```